### PR TITLE
Improve track_file handling and enable vrrp_track_file for keepalived < 2.1.0

### PR DESCRIPTION
**Pull request by bluthg**
#### Pull Request (PR) description
keepalived < 2.1.0 used (uses, it's e.g. in Ubuntu 20.04) "vrrp_track_file" instead of "track_file".
The PR uses a pragmatic approach to enabling this by simply cloning the type & code. Tested @ client, works as expected.

Also, the current implementation sets a hardcoded "weight" (to 1, which is the default anyway), but the "track_file" (and now "vrrp_track_file") has that as a parameter, so we should honour that.

And since the parameter is called "@param track_file" but actually wants an array, adds that to the description.
#### This Pull Request (PR) fixes the following issues
#278

### CODE REVIEW

**Comment by bluthg** [comment]
Linking to #278 and #281

**Comment by bastelfreak** [comment]
@bluthg can you please rebase against our latest master branch to get rid of the merge commits?

**Comment by bluthg** [comment]
> @bluthg can you please rebase against our latest master branch to get rid of the merge commits?

Uh oh, me and git...

File: manifests/vrrp/instance.pp [hunk 1]
```diff
@@ -205,6 +209,7 @@
   $track_script                                                           = undef,
   Optional[Array[String[1]]] $track_process                               = undef,
   Optional[Array[String[1]]] $track_file                                  = undef,
+  Optional[Array[String[1]]] $vrrp_track_file                             = undef,
```
Comment (bastelfreak) [inline manifests/vrrp/instance.pp:212]: Is that an array of absolte paths? then you can use:
```suggestion
  Optional[Array[Stdlib::Absolutepath] $vrrp_track_file                             = undef,
```

Comment (bluthg) [reply manifests/vrrp/instance.pp]: But then we should really use it for both types, no?

**Comment by bluthg** [comment]
And now the deletion of .puppet-lint.rc is in the MR again... sigh!

**Pull request merged**